server/controllers/routes/user/student/getAlumniCohort.js

```diff
@@ -0,0 +1,28 @@
+const { getSpecificAlumni } = require('../../../../database/queries');
+const { getCohortQuery } = require('../../../../database/queries');
+
+const getAlumniCohort = async (req, res, next) => {
+  try {
+    const { cohortId } = req.params;
+    if (cohortId > 0) {
```

Comment:
if cohortId is not a number will go throw else condition 
```js
else {
      res
        .status(404)
        .json({ statusCode: 404, message: 'You enterd wrong cohort ID' });
    }
```
beacuse this condition ( ``` if(cohortId > 0) ``` ) will work if cohort id is not number like this:
in console 
![daaa](https://user-images.githubusercontent.com/55782435/78452857-9bd7bb00-7696-11ea-850d-db555bc5f6af.png)
so it will go to else statment

this is a test by postman 
![147](https://user-images.githubusercontent.com/55782435/78452904-f1ac6300-7696-11ea-9abc-711b609ec5eb.png)